ipatests/azure/azure-pipelines.yml

```diff
@@ -20,12 +20,6 @@ jobs:
   steps:
     - template: templates/${{ variables.PREPARE_BUILD_TEMPLATE }}
```

Comment:
Wouldn't it work if we only call here:
- template: templates/${{ variables.PREPARE_LINT_TEMPLATE }}
and make no other change? This way we force installation of pylint respecting `--constraint .wheelconstraints.in` and we have the same version of pylint in this pipeline and in the "Lint" pipeline

Reply:
Yes, it would work.
But as I pointed out in https://github.com/freeipa/freeipa/commit/7855c77f93e0bf326e0307bce7072d5c12915659
pip can install everything needed for Pylint with the help of `--ignore-installed` (https://pip.pypa.io/en/stable/cli/pip_install/#cmdoption-I) even a dependency has been resolved in system site-packages.

Reply:
I think it's the primary goal of that option and that's ok for our usage.

Reply:
Not all IPA Python scripts use isolated mode, so it's maybe dangerous.